### PR TITLE
Fix Apple TV compile errors

### PR DESCRIPTION
**Pull request by dlowder-salesforce**
## Motivation

Apple TV compilation is currently broken in master -- this fixes it.

## Test Plan

Existing tests will pass after this change.

### CODE REVIEW

File: React/Modules/RCTUIManager.h [hunk 1]
```diff
@@ -148,8 +148,8 @@ RCT_EXTERN NSString *const RCTUIManagerRootViewKey;
 - (void)rootViewForReactTag:(NSNumber *)reactTag withCompletion:(void (^)(UIView *view))completion;
 
 /**
- * Finds a view that is tagged with {@param nativeId} as its nativeID prop
```
Comment (javache) [inline React/Modules/RCTUIManager.h]: Any reason to change these?

File: React/Modules/RCTDeviceInfo.m [hunk 2]
```diff
@@ -106,6 +106,7 @@ - (void)interfaceOrientationDidChange
 
 - (void)_interfaceOrientationDidChange
 {
+#if !TARGET_OS_TV
```
Comment (javache) [inline React/Modules/RCTDeviceInfo.m]: Can you instead hide the methods completely if `!TARGET_OS_TV`? So put them from line 96 to line 124.

File: React/Modules/RCTUIManager.h (hunk 1 above)
Comment (dlowder-salesforce) [inline React/Modules/RCTUIManager.h]: The original is giving a "documentation error"  when compiling and running the unit and integration tests.

File: React/Modules/RCTDeviceInfo.m (hunk 2 above)
Comment (dlowder-salesforce) [inline React/Modules/RCTDeviceInfo.m]: Done.

File: React/Modules/RCTUIManager.h (hunk 1 above)
Comment (dlowder-salesforce) [inline React/Modules/RCTUIManager.h]: compile error image

**Pull request closed without merging**